CHANGES.rst

```diff
@@ -36,8 +37,10 @@ Bugs
 ++++
 
 * ``First``, ``Rest`` and  ``Last`` now handle invalid arguments.
-* ``N`` now handles arbitrary precision numbers when the number of digits is not specified.  
-  
+* ``N`` now handles arbitrary precision numbers when the number of digits is not specified.
```

Comment:
Do the changes below do this? And the `SetTag` thing too?

Reply:
no, this is older. I think that the update is due a blank line...

Reply:
and yes, `Set*` is affected.